app/controllers/responsiblepeople/PersonNonUKPassportController.scala

```diff
@@ -59,8 +59,7 @@ class PersonNonUKPassportController @Inject()(
       rp <- getData[ResponsiblePerson](cache, index)
     } yield (rp.dateOfBirth.isDefined && edit) match {
       case true => Redirect(routes.DetailedAnswersController.get(index, flow))
-      case false if appConfig.phase2ChangesToggle => Redirect(routes.CountryOfBirthController.get(index, edit, flow))
-      case _ => Redirect(routes.DateOfBirthController.get(index, edit, flow))
```

Comment:
Is the removal of this line correct? Are we sure we never want to get to `DateOfBirthController`?

Reply:
Good spot, in the current flow from passport question people go to country of birth, however I think the line I've deleted was for safety reasons, to handle applications without the date of birth (when it was not mandatory), so I'll put it back, thanks!